docs/data/date-pickers/migration-lab/migration-lab.md

```diff
@@ -81,3 +81,60 @@ Which will transform the imports like this:
 
 Components of the Community Plan such as `<DatePicker />` can be imported from both `@mui/x-date-pickers-pro` and `@mui/x-date-pickers`.
 Only [date adapters](/x/react-date-pickers/getting-started/#setup) such as `AdapterDayjs` can only be imported from `@mui/x-date-pickers/[adapterName]`.
+
+### 3. Handle breaking changes introduced in alpha
+
+#### Props renaming
+
+The `disableCloseOnSelect` prop has been replaced by a new `closeOnSelect` prop which has the opposite behavior.
+The default behavior remains the same (close after the last step on desktop but not on mobile).
+
+```diff
+// If you don't want to close after the last step
+-<DatePicker disableCloseOnSelect={false} />
++<DatePicker closeOnSelect />
+
+// If you want to close after the last step
+-<DatePicker disableCloseOnSelect />
++<DatePicker closeOnSelect={false} />
+```
+
+The props of `MonthPicker`, `YearPicker` and `DayPicker` have been reworked to make them more consistent for a standalone usage.
+
+- **MonthPicker**: The prop `onMonthChange` has been removed, you can use `onChange` instead since every change is a month change.
+- **YearPicker**: The prop `onYearPicker` has been removed, you can use `onChange` instead since every change is a year change.
+- **DayPicker**: The prop `isDateDisabled` has been removed, you can now use the same validation props as for the other components (`maxDate`, `minDate`, `shouldDisableDate`, `disableFuture` and `disablePast`).
+
+#### Translation
+
+Props for translation have been either deprecated or removed in favor of a [global localization](/x/react-date-pickers/localization/) similar to the one used by the data grid.
+We already have nine translations provided by the community.
```

Comment:
I wanted to emphasise that translations are available and shared but not sure it's a good idea

Reply:
👍 Just maybe change the copy to be more in line with localisation? 🤔 
```suggestion
We already have nine locales provided by the community.
```

Reply:
On another note. Is this actually a typo? Or should it really say `DataGrid` here?
<img width="1313" alt="Screenshot 2022-08-17 at 09 28 40" src="https://user-images.githubusercontent.com/4941090/185050220-fa10ebef-8255-495f-817b-a8a56d118c33.png">

Reply:
No, it's a wrong copy/past